src/View/layout.html.twig

```diff
@@ -18,7 +18,7 @@
 		<body>
 			<nav class="navbar navbar-expand-lg">
 				<div class="container-fluid">
-					<a class="navbar-brand" href="#">
+					<a class="navbar-brand" href="/../home">
```

Comment:
The double dot are not needed, it should be /home

Reply:
Done !

Reply:
Nope

Reply:
:( Now it's ok !